nucliadb_paragraphs/src/query_io.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2021 Bosutech XXI S.L.
```

Comment:
I thought this conversation could come up. Although now it would be really easy to generalise those modules into one receiving a `Field` instead of an `Schema` I think is too soon to generalise the filtering logic of paragraphs and texts.
This filtering strategy that uses a query language is not mature enough and I feel will be prone to changes soon, so I prefer to wait for a bit before building a general solution.